Dispatcher/UI/Search/SearchScreenViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  SearchScreenViewController.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 02/05/2023.
+//
+
+import UIKit
+
+class SearchScreenViewController: UIViewController {
+
+    @IBOutlet weak var searchTableView: UITableView!
+    @IBOutlet weak var clearButton: UIButton!
+    @IBOutlet weak var babkButton: UIButton!
+    @IBOutlet weak var searchBar: UISearchBar!
+    
+    let searchScreenViewModel = SearchViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        customSearchBar()
+       
+        searchTableView.dataSource = self
+        searchTableView.delegate = self
+        searchTableView.register(UINib(nibName: AppConstants.latestSearchCellNibName, bundle: nil), forCellReuseIdentifier: AppConstants.latestSearchesCellIdentifier)
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        searchScreenViewModel.fetchLatestSearchs()
+        searchTableView.reloadData()
+    }
+    
+    private func customSearchBar() {
```

Comment:
not an informative function name - we can't know what does the function do unless we read its code